packages/vuetify/src/composables/autocomplete.ts

```diff
@@ -0,0 +1,34 @@
+// Utilities
+import { computed, ref } from 'vue'
+
+// Types
+export interface InputAutocompleteProps {
+  autocomplete?: 'suppress' | string
```

Comment:
Use `| undefined` not `?:` or it doesn't actually enforce that the prop is declared.